contrib/online_ide/sources/nit/pnacl_nit.nit

```diff
@@ -72,7 +72,15 @@ redef class FileReader
 		self.path = path
 		var file = sys.files[path]
 		prepare_buffer(file.length)
-		_buffer.append(file)
+		if path isa FlatString then
```

Comment:
a `copy_to_buffer` in Text can remove the `isa` and enable specific fast-path implementation in other subclasses of Text